README.md

```diff
@@ -100,6 +100,9 @@ This is list of known compatible USB hubs:
 | Raspberry Pi       | 3B+        ([see below](#raspberry-pi-3b))           | 4     | 2.0 |`0424:2514`| 2018    |      |
 | Raspberry Pi       | 4B         ([see below](#raspberry-pi-4b))           | 4     | 3.0 |`2109:3431`| 2019    |      |
 | Renesas            | uPD720202 PCIe USB 3.0 host controller               | 2     | 3.0 |           | 2013    |      |
+| Revolution Pi      | Core 3 / S / SE                                      | 2     | 2.0 |`0424:9514`| 2017    |      |
+| Revolution Pi      | Connect (+) / S / SE                                 | 2     | 2.0 |`0424:9514`| 2018    |      |
+| Revolution Pi      | Connect 4                                            | 2     | 3.2 |           | 2022    |      |
```

Comment:
I highly doubt that it actually supports USB 3.2 (Gen 2x2 20gbit/s). It is most likely 3.0 or possibly 3.1

Reply:
In the USB specs, they simplified the naming with USB 3.2. Thus 3.2 gen 1 is egual to 3.0 or 3.1. The official spec from RevPi Connect 4 is USB 3.2 gen1. I would be also fine with USB 3.0, so it's up to you

Reply:
Correct. But we decided to use older versioning scheme. With new standard, even first USB 3.0 devices are technically USB 3.2, which is not very useful.